engine/swarm/services.md

```diff
@@ -79,14 +79,16 @@ flag. For more information, see
 ### Specify the image version the service should use
 
 When you create a service without specifying any details about the version of
-the image to use, the service uses the latest version available in Docker Hub
-or your registry. You can force the service to use a specific version of
-the image in a few different ways, depending on your desired outcome.
+the image to use, the service uses the version tagged with the `latest` tag.
+You can force the service to use a specific version of the image in a few
+different ways, depending on your desired outcome.
 
 An image version can be expressed in several different ways:
 
-- If you specify a tag, the manager resolves that tag to a digest, and directs
-  workers to uses that version.
+- If you specify a tag, the manager resolves that tag to a digest. If you are
+  using [content trust](#image_resolution_with_trust), Notary verifies that the
```

Comment:
Maybe we should say "the docker client" instead of "Notary"?  Or "the docker client uses notary to verify..."?  It both verifies that it is signed and also resolves the tag to a digest.  The manager does not resolve the tag in this case (if content trust is enabled).

Also, wondering if we should link to https://docs.docker.com/engine/security/trust/content_trust/ when saying "signed" so users can look up more detail about how that works?